package.json

```diff
@@ -34,6 +34,7 @@
     "webpack-cli": "3.3.11"
   },
   "devDependencies": {
+    "husky": "^4.2.3",
```

Comment:
This is a new dependency but you haven't add any of the configuration values to enable the hooks. Is this dependency required?